kubewarden-load-policies.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
I would add `set -euo pipefail`. Given that we are introducing shell scripts, I would add a shellcheck CI job as we have in other repos.

Reply:
done in https://github.com/kubewarden/kwctl/pull/354/commits/1eb07dd70a233cdc5203b2befa13a2e5e8c10b89 and https://github.com/kubewarden/kwctl/pull/354/commits/25e00063b214877600c5ff9464b7c89d34bd3816